src/components/MessagesList/MessagesList.vue

```diff
@@ -1073,6 +1056,13 @@ export default {
 	flex: 1 0;
 	overflow-y: auto;
 	overflow-x: hidden;
+	border-bottom: 1px solid var(--color-border);
+	transition: border-bottom-color 150ms ease-in-out;
```

Comment:
Any reason not to use one of these?
https://github.com/nextcloud/server/blob/24ffe328452e0e247017894ea9d49944470fd2bc/apps/theming/lib/Themes/DefaultTheme.php#L166-L167

Reply:
To be sync with transition:fade effect. Could be replaced by variable though:
https://github.com/nextcloud/spreed/blob/915ab9d92b96d7e6b4dc8a6b769a743c8862d28f/src/assets/variables.scss#L100